install.sh

```diff
@@ -568,6 +570,7 @@ do_install() {
 					$sh_c "dnf makecache"
 				elif command_exists dnf; then
 					$sh_c "dnf -y -q --setopt=install_weak_deps=False install dnf-plugins-core"
+					$sh_c "rm -f /etc/yum.repos.d/docker-ce.repo  /etc/yum.repos.d/docker-ce-staging.repo"
```

Comment:
Not sure if there's  cleaner ways to do this; I couldn't find subcommands  or options in older `dnf` versions to replace existing repo-configs or to easily "reconcile" the options for each repo (without writing complex steps to generate the `baseurl` etc).